charts/vela-core/values.yaml

```diff
@@ -268,3 +268,9 @@ authentication:
   withUser: true
   defaultUser: kubevela:vela-core
   groupPattern: kubevela:*
+
+## @param sharding.enabled Enable sharding for core controller
+## @param sharding.shards The shards available for scheduling. If empty, dynamic discovery will be used.
+sharding:
+  enabled: false
```

Comment:
why don't make the default value to be true?

Reply:
what should one user do when they want to upgrade to the sharding mode when they don't enable it at first installation

Reply:
I noticed we need new role and rolebinding configuration, that might be hard for users to learn

Reply:
> why don't make the default value to be true?

By default, in the sharding mode, the master will not work on applications.

Reply:
> what should one user do when they want to upgrade to the sharding mode when they don't enable it at first installation

Just run `helm upgrade --set sharding.enabled=true`

Reply:
> I noticed we need new role and rolebinding configuration, that might be hard for users to learn

User does not need to know the role/rolebinding configuration.

Reply:
> By default, in the sharding mode, the master will not work on applications.

You should add that to the doc and KEP, I missed this message at first time.

Reply:
Now the master shard by default also accepts applications. So slaves are not required.